com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/message/Messages.java

```diff
@@ -174,6 +174,9 @@
     public static final String PURGE_DELETE_REQUEST_SPACE_FROM_CONFIGURATION_TABLES = "All delete request spaces after date: {0} will be deleted from configuration tables.";
     public static final String RETRIEVED_USER_TOKEN = "Retrieved token for user: {0} with expiration time: {1}";
     public static final String FSS_CACHE_UPDATE_TIMEOUT = "Fss cache update timeout: {0} minutes";
+    public static final String SHUTDOWN_REQUEST = "Application with id:\"{0}\", instance id:\"{1}\", instance index:\"{2}\", requested for shutdown at:\"{3}\" with cooldown timeout:\"{4}\" seconds.";

```

Comment:
In the 3 messages there is a cooldown timeout which is not meaningful enough. Maybe changing it to something which is more related to jobs, process engine?